pkgs/top-level/all-packages.nix

```diff
@@ -4805,6 +4805,11 @@ in
     inherit (darwin.apple_sdk.frameworks) Foundation;
   };
 
+  mono44 = callPackage ../development/compilers/mono/4.4.nix {
```

Comment:
I think we should make mono44 "lowPrio" for now, so that ```nix-env -i mono``` installs the same version as ```nix-env -iA mono```. (Without lowPrio, ```nix-env -i mono``` installs mono-4.4, which may seem odd when ```nix-env -iA mono``` installs mono-4.0.)